packages/models-library/src/models_library/services.py

```diff
@@ -1,30 +1,30 @@
-import sys
+"""
+
+NOTE: to dump json-schema from CLI use
+    python -c "from models_library.services import ServiceDockerData as cls; print(cls.schema_json(indent=2))" > services-schema.json
+"""
 from enum import Enum
-from pathlib import Path
 from typing import Dict, List, Optional, Union
 
 from pydantic import BaseModel, EmailStr, Extra, Field, HttpUrl, constr, validator
 from pydantic.types import PositiveInt
 
-from .constants import VERSION_RE
+from .basic_regex import VERSION_RE
 
-current_file = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve()
 
+SERVICE_KEY_RE = r"^(simcore)/(services)/(comp|dynamic|frontend)(/[^\s/]+)+$"
+KEY_RE = SERVICE_KEY_RE  # TODO: deprecate this global constant by SERVICE_KEY_RE
```

Comment:
hmm why not remove right away?

Reply:
I am cautions with the model. too many chagnes